### DISCUSSION TITLE
Latest

### DISCUSSION THREAD

**Discussion by Schwartzblat** [Schwartzblat/WhatsAppPatcher#70] [Announcements]
Please let me know if the ab tests feature works as expected.
It will probably crash the app because some of the new features hasn't been implemented yet and I will have to find the "bad tests" and blacklist them.

<hr /><em>This discussion was created from the release <a href='https://github.com/Schwartzblat/WhatsAppPatcher/releases/tag/1.0.13'>Latest</a>.</em>

**Comment by Turhvjbufv** [comment]
Sorry if I'm dumb but what are ab tests?

**Reply by Schwartzblat** [reply]
When a company wants to add a new feature to their product, they apply the change for some of the users to test their reaction to it.
The APK for every person is the same so usually, the server provides the list of features to enable.
I found this mechanism and activated all the in progress features to make the app look cooler and see what will be at WhatsApp beta in the next few months.
The problem is that some of the tests are crashing the app so I need to isolate them and disable them by default.
I hope this explanation helped you understand :)

**Reply by Turhvjbufv** [reply]
You explained it well, thank you
